cdap-common/src/main/java/co/cask/cdap/common/logging/LoggingContextAccessor.java

```diff
@@ -51,6 +51,13 @@ public static void setLoggingContext(LoggingContext context) {
   }
 
   /**
+   * Removes the logging context
+   */
+  public static void resetLoggingContext() {
+    loggingContext.remove();
```

Comment:
MDC itself is never updated with the logging context. addExtraTags() adds these tags directly to the event using event.getMDCPropertyMap().putAll(tags);